api/controllers/accounts.js

```diff
@@ -21,16 +21,23 @@ var swaggerHelper = require('../../helpers/swagger');
 var modules;
 
 /**
- * Initializes with scope content and private variables:
- * - modules
- * @class AccountsController
- * @classdesc Main System methods.
- * @param {scope} scope - App instance.
+ * @class
+ * @memberof controllers
+ * @requires lodash
+ * @requires helpers/apiError
+ * @requires helpers/swagger.generateParamsErrorObject
```

Comment:
The generated output seems to include this information twice now:
![screen shot 2018-01-25 at 12 33 46](https://user-images.githubusercontent.com/11036220/35386354-0d00eb2c-01cc-11e8-8c19-33a99c49cec8.png)

Reply:
Strange. At the moment, I cannot see where these 2 Requires come from. 
I thought it might already parse from the code itself, but if I remove the `@requires`-tag, it does not appear in the JSDocs at all.